security/src/main/resources/messages_en.properties

```diff
@@ -22,3 +22,15 @@ assign_permission_error_for_role=Unable to assign $ permission for $ role
 remove_permission_from_role_error=Unable to remove $ permission for the $ role
 remove_package_error=Unable to remove $ package from $ client
 role_remove_from_package_error=Unable to remove $ role from $ package
+client_password_policy_error=Given password is violating the policy which is defined by the client
```

Comment:
Password is violating the client's policy

Reply:
updated